testsuite/pytests/test_changing_tic_base.py

```diff
@@ -36,52 +36,6 @@ class TestChangingTicBase(unittest.TestCase):
     def setUp(self):
         nest.ResetKernel()
 
-    def test_models(self):
```

Comment:
Why are you removing this test? Or rather, why is it justified to do so?

Reply:
The test was testing that values are the same before and after changing the number of tics per ms and the resolution. This is not the case anymore for the models that are now automatically updating their values. On second thought, I should probably rather create a mechanism that just excludes auto-adjusted values from the check, instead of removing the test altogether.

Reply:
I've re-added the test extended by an ignore-list for automatically adjusted parameters in 6436b9811.